### ISSUE TITLE
Log Error - Unable to parse services.yaml for the emporia_vue integration

### ISSUE THREAD

**Issue by tanbronson** [magico13/ha-emporia-vue#191]
I'm running the latest version of the plug in.
Home Assistant 2023.4.1
Supervisor 2023.04.0
Operating System 9.5
Frontend 20230406.1 - latest

I'm having a separate issue with a broken zigbee connection with my power meter, but the app and the integration both show zero values (so they appear to match -)

I did try to reinstall the integration, and the message didn't go away

**Comment by magico13** [comment]
I also see that in my logs on 2023.4, though the one existing service for the EVSE appears to still be functioning. I'll have to see what changed for it to give that message but it doesn't appear to be affecting anything on my setup.

**Comment by Aipoc76** [comment]
same error. cant see anything not working thou.

```
Logger: homeassistant.helpers.service
Source: helpers/service.py:553
First occurred: 9:28:11 AM (1 occurrences)
Last logged: 9:28:11 AM

Unable to parse services.yaml for the emporia_vue integration```

**Comment by skynet01** [comment]
Any updates on this? Is it safe to ignore this one?

**Comment by magico13** [comment]
It appears to be pretty safe to ignore for now. There's a PR to fix it that I haven't had a chance to pull in and test yet.

**Comment by magico13** [comment]
Resolved with version 0.8.2
